superbit_lensing/shear_profiles/make_annular_catalog.py

```
@@ -399,26 +383,52 @@ def main(args):
     overwrite = args.overwrite
     vb = args.vb
 
+
+    # Define position args                                                                                                                          
+    xy_cols = ['X_IMAGE', 'Y_IMAGE']
+    shear_args = ['g1_Rinv', 'g2_Rinv']
+    
+    ## Get center of galaxy cluster for fitting
+    ## Throw error if image can't be read in
+    
+    try:
+        coadd_im_name = os.path.join(outdir, f'{run_name}_mock_coadd.fits')
+        assert os.path.exists(coadd_im_name) is True
+        hdr = fits.getheader(coadd_im_name)
+        xcen = hdr['CRPIX1']; ycen = hdr['CRPIX2']
+        coadd_center = [xcen, ycen]
+        print(f'Read image data and setting image NFW center to ({xcen},{ycen})')
+
+    except:
+        print('\n\n\nNo coadd image center found, cannot calculate tangential shear\n\n.')
```

This should raise an error instead of just printing the problem.

---

done